leo/src/DAO/supabase/SupabaseEventsDAO.ts

```diff
@@ -1,22 +1,104 @@
-import {EventsDAO} from "@/DAO/interface/EventsDAO";
-import {createClient} from "@/utils/supabase/server";
-import {Tables} from "../../../types/database.types";
+import { EventsDAO } from "@/DAO/interface/EventsDAO";
+import { createClient } from "@/utils/supabase/server";
+import { Tables, TablesInsert, TablesUpdate } from "../../../types/database.types";
 
 export class SupabaseEventsDAO implements EventsDAO {
     private supabase = createClient();
     private TABLE = 'Events'
+    private BUCKET = 'events-pictures'
```

Comment:
i wonder if we should have things like this in a config file somewhere? do you know how to do that